scripts/profile-compiler-build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# To be run from the root of the Flambda backend repo
+
+set -e -u -o pipefail
+
+dump_dir="`pwd`/profile"
```

Comment:
(May use `_profile` rather than
`profile`. That conveys the idea
that, like `_build` or `_runtest`
the folder contains artifacts
produced during the build.
Perhaps even add `_profile` to
`.gitignore`.)